conf/application.conf

```diff
@@ -149,3 +144,59 @@ microservice {
         }
     }
 }
+
+
+# Microservice specific config
+
+wiremock-port = 11111
+wiremock-port = ${?WIREMOCK_PORT}
+
+# TODO discover if this is needed
```

Comment:
In this file (see whole content), remove all empty configurations if not used:
- microservice
- services
- Stub

Reply:
done